ibm/mas_devops/roles/kubeturbo/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Define where Kubeturbo will be installed, and to which Turbonomics it will connect to
+turbonomic_route: "{{ lookup('env', 'TURBONOMIC_ROUTE') }}"
+turbonomic_password: "{{ lookup('env', 'TURBONOMIC_PASSWORD') }}"
+turbonomic_username: "{{ lookup('env', 'TURBONOMIC_USERNAME') }}"
+kubeturbo_namespace: "{{ lookup('env', 'KUBETURBO_NAMESPACE') | default('kubeturbo', true) }}"
+kubeturbo_sub_name: "{{ lookup('env', 'KUBETURBO_SUBNAME') | default('kubeturbo-certified', true) }}"
+kubeturbo_channel: "{{ lookup('env', 'KUBETURBO_CHANNEL') | default('stable', true) }}"
+kubeturbo_cr_name: "{{ lookup('env', 'KUBETURBO_CR_NAME') | default('kubeturbo-release', true) }}"
+kubeturbo_tag_release: "{{ lookup('env', 'KUBETURBO_TAG_RELEASE') | default('8.7.5', true) }}"
+cluster_name: "{{ lookup('env', 'CLUSTER_NAME') }}"
```

Comment:
what is CLUSTER_NAME? is it the OCP Cluster name or something else? If it is something else then a different variable name might be needed.

Reply:
Yes, the CLUSTER_NAME is the OCP Cluster name.